### DISCUSSION TITLE
Prominence of the MUNify brand identity

### DISCUSSION THREAD

**Discussion by Amselfass** [DeutscheModelUnitedNations/munify-delegator#90] [Ideas & Feature requests]
I'm not sure the „enhancement“ label really fits the point I want to make, but it seemed the most appropriate out of the bunch.

I have noticed that the app, in its current state, is trying very hard to establish _MUNify_ and its subprojects as its own, distinct brand identity. This is evident from many aspects:

- The names _MUNify_, _MUNify DELEGATOR_, _MUNify GUARD_ etc. are featured very prominently in logos and UI texts. This is a major change from its predecessor _MUNdi_, whose name was never used in outside communications. To our participants, _MUNdi_ was always just known as „the MUN-SH / MUNBW website“.
- The app leans heavily into the [Corporate Memphis](https://en.wikipedia.org/wiki/Corporate_Memphis) artstyle, which is not and has never been used anywhere else at DMUN and makes the app feel removed from the rest of the content at mun-sh.de / munbw.de.
- MUN-SH / MUNBW and their respective logos play a very minor role in the app's design and UI, despite being DMUN's two strongest brand identities and arguably the only ones with a noteworthy recognition value amongst our target audience.

To an outside observer, the app gives off the vibes of a white-label platform which DMUN has purchased or licensed, and does not look like our in-house development. I'd argue it would strengthen the brand perception of DMUN, MUN-SH and MUNBW to remove most of the _MUNify_ branding and lean more heavily into our established brands.

Purely from a DMUN perspective, I see little point in trying to establish _MUNify_ as a new brand identity with our participants and their teachers, when even the DMUN brand itself is still so heavily overshadowed by MUN-SH and MUNBW.

I'm opening this issue in case this has not been a conscious decision by the developers. Of course, a reasoning like „MUNify needs its separate brand identity so that @TadeSF and @m1212e can use it in their portfolio for their future careers“ would also be an entirely valid reason.

(I know it's too early in the development to discuss artwork and graphics, but since the issue is relevant for future UI wording efforts, I wanted to get it on the books early.)

**Comment by m1212e** [comment]
I do see your point. The decision was only partially a conscious decision afaik. In theory this project can and should live not only in the DMUN universe but be available to all kinds of MUNs which need some kind of software to run their conference, therefore there's the munify "brand" which originally was just a name for our endeavor to digitalize our conferences and everything around them. I think this discussion is one we should have in-house, me personally I don't really have a strong oppinion on this. But as you correctly mentioned, this probably is not a thing we can dedicate much resources to, as long as the app is not in a state which can be somehow called "done". Maybe this is something for our november "Sommerfest"? :D

**Comment by Strehk** [comment]
Thanks @Amselfass for raising your concerns. I agree with what @m1212e stated, but I wanted to stress and expand on a few points further:
As @m1212e already stated, one of our biggest motivation was not only, that we wanted to provide DMUN with more up to date digital infrastructure, but also very much the concern, that other MUNs are being driven to use proprietory and costly software to run their conferences, which the sellers of this software do not only as a non-profit endevour, but rather to actually gain profit from it. We deeply believe that this kind of software should be free – or at least be priced to only cover running costs (servers, etc.). This is why we call MUNify first and foremost an Open Source Initiative, that is ofcourse aided and primarily used by DMUN but not developed exclusively for – like MUNdi was.
By the way, this is not the first time that software is primarily developed for DMUN but not itended for exclusive use: MUNity was the same and only adopted by DMUN after the maintainer pulled himself out of the project.
The Prefix and  "corporate brand name" MUNify plays (in my opinion) a crucial role in making this transparent:
1. You were talking about the fact, that it appears detached from our websites and our identiy. That – in a way – was one of our goals. We did want to create software that is universally usable. Tailored ofcourse to our most prominent use case (DMUN), but still in a way white lable enough to enable other conferences to adopt.
2. By creating this software suite with its own brand name, we enable participants, that may later move to other conferences as chairs or organizers to spread the word about our software. We actually want MUNify to have a strong individual identity, because we want to see MUNify be used in many other conferences.
3. A unified Brand name in general plays also a big role in communicating that it is not (necessarily) standalone software. Take CHASE for example – you can absolutely use CHASE on its own. But our vision is, that a conference might think: "hey, if we are using CHASE already for our on conference stuff, let's make our lives easier and also use the DELEGATOR for registration, since it is well connected with CHASE. This is especially important since we are not using a monolithic approach: Our software is seperated into smaller parts that you – as a conference – could opt-in to use at any point.
4. We also want to make transparent that MUNify is adaptable and not purely for DMUN use. That is why we have a truly open source licens (AGPL) in all our projects related to MUNify. We are more then happy to accept forks and help out in adapting the software to fit the needs of other conferences better.

I see you concern that it feels "white lable" and we can definetly talk about making DMUN itself more prominent in CHASE and DELEGATOR and whats to come. But I would strongly advocate against binding the software closely to DMUN as you suggested. That would undermine one of my main goals developing these apps: Staying open and neutral for any (best non-commercial) use by other conferences and initiatives with the same goals: Enabeling democracy supporting projects like MUNs.

Btw, that is also the reason why we keep all of our on-Github-communication english – we want everyone to be able to participate/fork and work on their own version.

I hope that clarifies why we have taken this approach. It is of course also nice for our portfolios, but that has never been a driving motive ;)

Also a little sidenote: The logos are still in development, @frederikschissler will probably overhaul all MUNify Logos to make them feel more like a product suite and also belong a little bit more to DMUN. Right now, these are just plain icons from an icon library that i decided on for our use.

**Comment by Amselfass** [comment]
Thanks for the thorough explanation. I share your vision to establish DMUN as a community hub for open-source MUN software, and I will keep this in mind when working on the rest of the UI text.

I have no problem with closing this discussion, should you so desire.
